frontend/src/App.vue

```diff
@@ -251,6 +251,12 @@ export default {
       }
       this.doneMessage = message
       this.isDone = true
+
+      const promise = new Promise((resolve, reject) => {
+        this.resolve = resolve
+        this.reject = reject
+      })
+      return promise
```

Comment:
How about `return Promise.resolve()`?

Don't we need to do this for every rpc-registered method? Can we somehow handle this in the `webview-rpc` library?

Reply:
I made a change in view-rpc (https://github.com/SAP/vscode-webview-rpc-lib/pull/5) that enables not returning a value

Until that pull request is merged and published to npm, it should be sufficient to return a non-undefined value (e.g. `{}`).